frontend/src/scenes/data-management/comments/Comments.tsx

```diff
@@ -0,0 +1,189 @@
+                        >
+                            {(comment.content ?? '').slice(0, 47) + '...'}
+                        </Tooltip>
+                    )
+                }
+                return <div className="font-semibold">{renderedContent}</div>
+            },
+        },
+        {
+            title: `Date and time (${shortTimeZone(timezone)})`,
+            dataIndex: 'created_at',
+            render: function RenderCreatedAt(_, comment: CommentType): string {
+                return dayjs(comment.created_at).format('MMM DD, YYYY h:mm A')
+            },
+            sorter: (a, b) => dayjs(a.created_at).diff(dayjs(b.created_at)),
+        },
+        {
+            title: 'Scope',
+            key: 'scope',
+            render: function RenderScope(_, comment: CommentType): JSX.Element {
+                return <LemonTag className="uppercase">{comment.scope}</LemonTag>
+            },
+        },
+        {
+            title: 'Created by',
+            dataIndex: 'created_by',
+            render: function RenderCreatedBy(_: any, comment: CommentType) {
+                return (
+                    <div className="flex flex-row items-center">
+                        <ProfilePicture user={comment.created_by} showName size="md" type="person" />
+                    </div>
+                )
+            },
+            sorter: (a, b) =>
+                (a.created_by?.first_name || a.created_by?.email || '').localeCompare(
+                    b.created_by?.first_name || b.created_by?.email || ''
+                ),
+        },
+        {
+            key: 'actions',
+            width: 0,
+            render: function RenderActions(_, comment: CommentType): JSX.Element {
+                const canEdit = user?.uuid === comment.created_by?.uuid
+                return (
+                    <div className="flex">
+                        {canEdit && (
+                            <>
+                                <LemonButton
+                                    icon={<IconTrash />}
+                                    size="small"
+                                    status="danger"
+                                    onClick={() => deleteComment(comment.id)}
+                                    disabledReason={canEdit ? undefined : 'You can only delete your own comments'}
+                                    data-attr="comment-management-scene-delete"
+                                />
+                                <LemonButton
+                                    icon={<IconOpenInApp />}
+                                    size="small"
+                                    to={openURLFor(comment) || ''}
+                                    disabledReason={
+                                        openURLFor(comment)
+                                            ? undefined
+                                            : 'Not able to link to this comment type 🙈- tell us you want to be able to!'
```

Comment:
```suggestion
                                            : 'Not able to link to this comment type 🙈. tell us you want to be able to!'
```

or skip the . altogether, i think its nicer